<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [DemoSuite]" guice-stage="DEVELOPMENT">
  <listeners>
    <listener class-name="GlobalexceptionHandler.GlobalExceptionHandler"/>
    <listener class-name="sampleTestNG.TestSuit.configurationsetup"/>
  </listeners>
  <test thread-count="5" name="OrangeHRMLogin(failed)">
    <parameter name="password" value="admin123"/>
    <parameter name="username" value="Admin"/>
    <classes>
      <class name="sampleTestNG.TestSuit.PIMmodule">
        <methods>
          <include name="AfterTestCompletion"/>
          <include name="AddNewEmployee"/>
          <include name="LoadApplication"/>
        </methods>
      </class> <!-- sampleTestNG.TestSuit.PIMmodule -->
    </classes>
  </test> <!-- OrangeHRMLogin(failed) -->
</suite> <!-- Failed suite [DemoSuite] -->
